Lib/test/test_typing.py

```diff
@@ -3172,6 +3172,9 @@ def meth(self): pass
         self.assertNotIn("__callable_proto_members_only__", vars(NonP))
         self.assertNotIn("__callable_proto_members_only__", vars(NonPR))
 
+        self.assertEqual(get_protocol_members(P), frozenset({"x"}))
+        self.assertEqual(get_protocol_members(PR), frozenset({"meth"}))
```

Comment:
frozensets and sets compare equal:

```pycon
>>> {"x", "y"} == frozenset({"x", "y"})
True
```

```suggestion
        self.assertEqual(get_protocol_members(P), {"x"})
        self.assertEqual(get_protocol_members(PR), {"meth"})

        # the returned object should be immutable,
        # and should be a different object to the original attribute
        # to prevent users from (accidentally or deliberately)
        # mutating the attribute on the original class
        self.assertIsInstance(get_protocol_members(P), frozenset)
        self.assertIsNot(get_protocol_members(P), P.__protocol_attrs__)
        self.assertIsInstance(get_protocol_members(PR), frozenset)
        self.assertIsNot(get_protocol_members(PR), P.__protocol_attrs__)
```